Assignment 3-JS/script/script.js

```diff
@@ -0,0 +1,320 @@
+
```

Comment:
Add document ready function and move everything inside that function.

Reply:
Moved everything inside document ready function except onclick functions.